src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.helpers.ts

```diff
@@ -174,16 +159,54 @@ export class DatasetFlowTableHelpers {
         return "";
     }
 
-    public static waitingBlockText(startCondition: MaybeNull<FlowStartCondition>, datasetKind: DatasetKind): string {
+    public static durationBlockText(node: FlowSummaryDataFragment): string {
+        const descriptionTypename = node.description.__typename;
+        switch (descriptionTypename) {
+            case "FlowDescriptionDatasetPollingIngest":
+            case "FlowDescriptionDatasetPushIngest": {
```

Comment:
PushIngest will never have a schedule, it can only be an external trigger

Reply:
Fixed